### DISCUSSION TITLE
How to add a new list of non-objects?

### DISCUSSION THREAD

**Discussion by irishtim1393** [textadventures/quest#1287] [Creating games with Quest] [answered]
I've been trying to add a feature where the player can type "abilities" and that will display a list of learned abilities so far. I haven't been able to create the list properly, it seems. I used the setvariable command to new string as the first step in the script that plays once loading into the starting room. Is this wrong?

**Comment by Dragonoftheworld** [comment, marked as answer]
Can you navigate through code view?
if so: 
go to player object and add attribute "abilities". Set the drop menu to stringlist.
Whenever the player learns a new ability, write:
```
list add (player.abilities, "kick ass")
```
make a new command, type "abilities" in its pattern. In its script type:
```
msg ("Your learnt abilities are: ")
foreach (x, player.abilities){
  msg (x)
}
```

**Reply by irishtim1393** [reply]
Thank you very much! This is just what I was looking for
